arcgis-ios-sdk-samples/Augmented reality/Display scenes in tabletop AR/README.md

```diff
@@ -17,9 +17,9 @@ You'll see a feed from the camera when you open the sample. Tap on any flat, hor
 1. Create an `ArcGISARView` and add it to the view.
     * Note: this sample uses content in the WGS 84 geographic tiling scheme, rather than the web mercator tiling scheme. Once a scene has been displayed, the scene view cannot display another scene with a non-matching tiling scheme. To avoid that, the sample starts by showing a blank scene with an invisible base surface. Touch events will not be raised for the scene view unless a scene is displayed.
 2. Listen for ARKit tracking state updates with `arView.arSCNViewDelegate` and provide feedback to the user as necessary.
-3. When tracking is ready, wait for the user to tap, then use `arView.setInitialTransformation(using: screenPoint)` to set the initial transformation, which allows you to place the scene. This method uses ARKit's built-in plane detection.
+3. When tracking is ready, wait for the user to tap, then use `arView.setInitialTransformation(using:)` to set the initial transformation, which allows you to place the scene. This method uses ARKit's built-in plane detection.
 4. Create and display the scene. To allow you to look at the content from below, set the base surface navigation constraint to `none`.
-5. To create a realistic tabletop mapping experience, set the scene's base surface opacity to 0. This will ensure that only the target scene content is visible.
+5. Set the clipping distance property of the AR view. This will clip the scene to the area you want to show.
```

Comment:
Though the README changes, what if the original code remains the same?

The current implementation still reflects the original design. Without touching the code, it will lead to discrepancy between readme and implementation.

Reply:
Excellent point. If we come across cases where the standardized README doesn't reflect the state of the code, then we shouldn't introduce that aspect of the README's change. But as much as possible, the rest of the README's standardized changes should still be made.